modules/imgproc/src/geometry.cpp

```diff
@@ -698,23 +698,25 @@ static Rect pointSetBoundingRect( const Mat& points )
         return Rect();
 
 #if CV_SIMD // TODO: enable for CV_SIMD_SCALABLE, loop tail related.
-    const int64_t* pts = points.ptr<int64_t>();
-
+    union { int64_t vali64; int32_t vali32[2]; float valf32[2]; } firstval;
     if( !is_float )
     {
+        const int32_t* pts = points.ptr<int32_t>();
+        firstval.vali32[0] = pts[0];
```

Comment:
I'm not sure if using `std::memcpy` instead of union would be better in this case:
```
memcpy(&vali64, pts, 2 * sizeof(pts[0]));
```